akvo/rsr/tests/results_framework/test_cumulative_updates.py

```diff
@@ -72,6 +74,9 @@ def setUp(self):
         })
         aggregate(self.period2.object)
 
+        self.period3 = self.project.get_period(period_start=self.PERIOD_3_START)
+        aggregate(self.period3.object)
```

Comment:
I'm more or less in line with you. This call needs to be triggered by the system whereas the others are triggered on indicator update changes. The problem is how and when the system triggers it.